mysql-test/include/log_encrypt_2.inc

```diff
@@ -0,0 +1,129 @@
+# InnoDB transparent tablespace data encryption
+# This test case will test basic encryption support features.
+
+--source include/no_valgrind_without_big.inc
+--source include/have_innodb.inc
+--source include/not_embedded.inc
+
+CREATE TABLE t1(c1 int) ENGINE=InnoDB ENCRYPTION="Y";
+
+DROP TABLE t1;
+
+# Restart the server with keyring loaded
+--let restart_parameters="restart:$KEYRING_PARAMS"
+--source include/restart_mysqld_no_echo.inc
+
+let $innodb_file_per_table = `SELECT @@innodb_file_per_table`;
+
+SET GLOBAL innodb_file_per_table = 1;
+SELECT @@innodb_file_per_table;
```

Comment:
Two very redundant lines, better to use assert.inc or nothing. OK to fix after the merge or never.